From: Arnaud Ferraris <aferraris@debian.org>
Date: Wed, 16 Aug 2023 23:02:48 +0200
Subject: arm64: dts: rk3399-pinephone-pro: Fix adc-keys thresholds

The "Volume up" button shorts the reference voltage to the ground, so
the voltage read can go as low as 0. On the other hand, "Volume down"
causes the voltage read to be around 300mV, sometimes a bit less.

Change the thresholds so they better match reality.

Signed-off-by: Arnaud Ferraris <arnaud.ferraris@gmail.com>
---
 arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts b/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
index 664c3ab..e5ce7b2 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
@@ -42,13 +42,13 @@ adc-keys {
 		button-up {
 			label = "Volume Up";
 			linux,code = <KEY_VOLUMEUP>;
-			press-threshold-microvolt = <100000>;
+			press-threshold-microvolt = <0>;
 		};
 
 		button-down {
 			label = "Volume Down";
 			linux,code = <KEY_VOLUMEDOWN>;
-			press-threshold-microvolt = <600000>;
+			press-threshold-microvolt = <250000>;
 		};
 	};
 
